ultradns-rest/src/main/java/denominator/ultradns/UltraDNSRestGeoSupport.java

```diff
@@ -0,0 +1,242 @@
+package denominator.ultradns;
+
+import java.util.Collection;
+import java.util.Iterator;
+import java.util.ArrayList;
+import java.util.Set;
+import java.util.Map;
+import java.util.TreeMap;
+import java.util.TreeSet;
+
+import javax.inject.Named;
+
+import dagger.Module;
+import dagger.Provides;
+import denominator.ultradns.model.DirectionalGroup;
+import denominator.ultradns.model.Region;
+import denominator.ultradns.util.RRSetUtil;
+import denominator.ultradns.util.RegionUtil;
+import org.apache.commons.lang.StringUtils;
+
+@Module(injects = UltraDNSRestGeoResourceRecordSetApi.Factory.class, complete = false)
+public class UltraDNSRestGeoSupport {
```

Comment:
So this single class is doing a lot of stuff, better to refactor this class and break into multiple logical classes.